metering/metering-installing-metering.adoc

```diff
@@ -5,10 +5,25 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+Before installing metering into your cluster, review the following sections.
+
+[WARNING]
+====
+Installing Metering on an Openshift FIPS-enabled cluster with TLS enabled will fail while attempting to generate TLS certificates for the user.
+
+See link:https://bugzilla.redhat.com/show_bug.cgi?id=1787745[BZ#1787745] for details.
+====
+
 On {product-title} 4 clusters, metering can be installed through OperatorHub. If you are using an earlier version of {product-title} or a Kubernetes cluster without OpenShift, use the link:https://github.com/operator-framework/operator-metering/tree/master/Documentation[upstream documentation] to install, configure, and use metering.
```

Comment:
s/OpenShift/{product-title}

Reply:
Did you get that GitHub link approved by support? There's a blanket ban on them without direct approval from support, and this feels like one that Eric would vote to pull.

Reply:
I don't think this link is necessary. Initially I was worried, but I think I can remove the link and just clarify that Metering is supported on 4.x and above cluster.